awdmcguinness/really-cool-team-project
# feat: Implement chat functionality for each subway line

### Description
Adds working chat functionality for each subway line

live link: https://subway-stories-3e972.web.app/
screenshot:
<img width="700" alt="Screen Shot 2019-12-17 at 8 13 43 PM" src="https://user-images.githubusercontent.com/54907095/71047480-c3b40a80-2109-11ea-8774-6a9b5507ffef.png">

### Steps to test
They need to follow the tutorial for installing `firebase-cli`:
https://firebase.google.com/docs/cli

Once firebase-cli is setup complete the following steps to test the app:
1. From the main project directory run `cd functions && npm install` on the command line
2. Then run `firebase serve` and open `localhost:5000` in your default browser
3. Ensure that weather API and tweets are displayed on the homepage properly
4. Select a line and direction
5. Ensure that the user is redirected to the proper chat
6.  Try to send a message and ensure the user is presented with the `You must sign-in first` toast/notifcation at bottom
7.  Sign-in with gmail and ensure that you can post a message

### Acceptance criteria
- [ ] The user should not be able to send a message when they are logged out
- [ ] The user should ...
